config.toml

```diff
@@ -14,3 +14,7 @@ OPENAI = "<YOUR_OPENAI_API_KEY>"
 
 [API_ENDPOINTS]
 BING = "https://api.bing.microsoft.com/v7.0/search"
+OPENAI_BASE_URL = "https://api.openai.com/v1"
+
+[USER_PREFS]
+SEARCH_ENGINE = "Bing" # Possible Values: Bing, DDGS. Default Value: Bing
```

Comment:
**Code Structure Issue**: The addition of the OPENAI_BASE_URL and the SEARCH_ENGINE settings under [USER_PREFS] introduces new configuration options but lacks documentation or comments explaining their usage or impact. <br> **Fix**: Add comments above these settings to explain their purpose, how they can be modified, and the effect they have on the system's behavior. This will improve the maintainability of the configuration file and make it easier for new developers to understand the purpose of these settings. <br> **Code Suggestion**: 
 ```
 +# OPENAI_BASE_URL: Specifies the base URL for the OpenAI API. Modify as needed based on the API version in use.
 +OPENAI_BASE_URL = "https://api.openai.com/v1"
 +
 +# SEARCH_ENGINE: User's preferred search engine. Possible Values: Bing, DDGS. Default Value: Bing
 +SEARCH_ENGINE = "Bing"
 ```